README.md

```diff
@@ -100,15 +100,15 @@ Here is a description of the events:
 import CallingExtensions from "@hubspot/calling-extensions-sdk";
 
 const options = {
-  // Whether to log various inbound/outbound messages to console
+  /** @property {boolean} debugMode - Whether to log various inbound/outbound debug messages to the console. If false, console.debug will be used instead of console.log */
```

Comment:
I think I saw more JS docs in this PR earlier. I really liked them and think they should be added to [the constructor](https://github.com/HubSpot/calling-extensions-sdk/blob/master/src/CallingExtensions.js#L10) so that they show up in the code editor, which is super handy.

Long term, I would love for this "create an instance" section to resemble [the Mozilla docs](https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Global_Objects/Map/Map) or [the new React docs](https://react.dev/reference/react/useState), which break up the documentation into sections like syntax/reference, parameters, and usage/examples.